znframe/frame.py

```diff
@@ -91,6 +92,10 @@ def from_atoms(cls, atoms: ase.Atoms):
             pbc=atoms.pbc,
             cell=atoms.cell,
         )
+        try:
+            frame.info["calc"] = atoms.calc.results
+        except AttributeError:
+            pass
 
```

Comment:
There will be duplicates here. If you check in `self.arrays` and `self.info` you'll see the same data as in `calc.results`.